pyrobosim/test/utils/test_pose_utils.py

```diff
@@ -135,32 +170,26 @@ def test_pose_to_from_dict():
     pose_dict = {
         "position": {"x": 1.0, "y": 2.0, "z": 3.0},
         "rotation_eul": {
-            "yaw": np.pi / 2.0,
-            "pitch": np.pi / 4.0,
-            "roll": -np.pi / 2.0,
+            "yaw": angle_value,
+            "pitch": angle_value / 2,
+            "roll": -angle_value,
         },
+        "angle_units": angle_units,
     }
     pose = Pose.from_dict(pose_dict)
     assert pose.x == pytest.approx(1.0)
     assert pose.y == pytest.approx(2.0)
     assert pose.z == pytest.approx(3.0)
-    assert pose.eul == pytest.approx([-np.pi / 2.0, np.pi / 4.0, np.pi / 2.0])
+    assert pose.eul == pytest.approx(expected_eul)
+    assert pose.angle_units == angle_units
 
-    pose_dict = {
-        "position": {"x": 1.0, "y": 2.0, "z": 3.0},
-        "rotation_eul": {
-            "yaw": np.pi / 2.0,
-            "pitch": np.pi / 4.0,
-            "roll": -np.pi / 2.0,
-        },
-        "rotation_quat": {"w": 0.707107, "x": -0.707107, "y": 0.0, "z": 0.0},
-    }
+    pose_dict["rotation_quat"] = {"w": 0.707107, "x": -0.707107, "y": 0.0, "z": 0.0}
     pose = Pose.from_dict(pose_dict)
     assert pose.x == pytest.approx(1.0)
     assert pose.y == pytest.approx(2.0)
     assert pose.z == pytest.approx(3.0)
-    assert pose.eul == pytest.approx([-np.pi / 2.0, 0.0, 0.0])
-    assert pose.q == pytest.approx([0.707107, -0.707107, 0.0, 0.0])
+    assert pose.eul == pytest.approx([-np.pi / 2, 0.0, 0.0])
+    assert pose.q == pytest.approx(expected_q)
```

Comment:
seems like `expected_q` is the same in both cases, so maybe no need to put this one in the `parametrize`?